### ISSUE TITLE
Append method?

### ISSUE THREAD

**Issue by jakearchibald** [jakearchibald/byte-storage#3]
```js
const writable = await byteStorage.append(name:String);
```

Same as `.write`, where `start` is equal to the store's size.

**Comment by dominictarr** [comment]
yes please! it may also be useful to be able to put a byte-store into `append-only` mode, which would disallow random writes, to rewrite you'd have to resize the file (aka, [truncate](https://www.gnu.org/software/libc/manual/html_node/File-Size.html)) and then write again.
